editoast/src/views/stdcm_search_environment.rs

```diff
@@ -241,7 +318,7 @@ pub(in crate::views) async fn list(
         .order_by(|| StdcmSearchEnvironment::ID.asc());
     let (listed_envs, stats) = StdcmSearchEnvironment::list_paginated(&mut conn, settings).await?;
     Ok(Json(SdcmSearchEnvListResponse {
-        results: listed_envs,
+        results: listed_envs.into_iter().map(|env| env.into()).collect(),
```

Comment:
minor: Since we already have `itertools` in our dependencies, you should be able to use `.map_into()`


```suggestion
        results: listed_envs.into_iter().map_into().collect(),
```